docs/man/rpm-man-template.scd

```diff
@@ -0,0 +1,70 @@
+RPMCOMMAND(1)
+
+# NAME
+rpmcmd - Example rpm man page
+
+# SYNOPSIS
+*rpmcmd* [options] [_ARGUMENT_] ...
+
+# DESCRIPTION
+Describe what *rpmcmd* command does in detail.
+
+Put distinct operational modes to their own subsections.
+
+Typographic conventions:
+- Use bold for literal values such as command name and options:
+*cmd* does X, with option *--do-this* does Y instead
+- Use underline uppercase to denote user arguments: _ARGUMENT_
+- Use underline for literal file names: _hello-1.0-1.noarch.rpm_
+- Put an empty line before a new section/option/etc
+- Put text immediately below a section/option/etc (ie without an empty line)
+- Put optional elements into [] brackets
+- Put one-of-many choices into <val1|val2|val3> lists
+- Do not emphasize separators like ",", "=", "[]", "|"
+
+When in doubt, consult *man-pages*(7).
+
+# OPTIONS
+Describe common options and in this section.
+
+Typographic conventions:
+- Put option aliases on separate lines
```

Comment:
Oh, perhaps this is what I suggested above :sweat_smile: Nevermind then. In that case, maybe just mention that the short alias should go first, to keep some level of consistency?

Reply:
Yeah, it's a good point. Short goes first, "obviously", but I see now I actually violated that rule in the examples :flushed: